app/views/user_mailer/email_changed.html.slim

```diff
@@ -12,4 +12,5 @@ table.hr
 
 p == t('.help',
       app: link_to(APP_NAME, Figaro.env.mailer_domain_name, class: 'gray'),
-      contact_phone_number: CONTACT_PHONE_NUMBER)
+      help_link: link_to(t('user_mailer.help_link_text'), MarketingSite.help_url),
+      contact_link: link_to(t('user_mailer.contact_link_text'), MarketingSite.contact_url))
```

Comment:
Not sure if this will work, but what if each mailer called the partial with a local variable that corresponds to the translation key. Then, in the partial, you can do something like `t("#{translation_key}.help)`?